compose/cli/main.py

```diff
@@ -875,7 +874,7 @@ def up(self, options):
         If you want to force Compose to stop and recreate all containers, use the
         `--force-recreate` flag.
 
-        Usage: up [options] [SERVICE...]
+        Usage: up [options] [--scale SERVICE=NUM...] [SERVICE...]
```

Comment:
I think we have to declare it separately because it's repeatable. Same goes for build for example:
`        Usage: build [options] [--build-arg key=val...] [SERVICE...]`

or run:
`        Usage: run [options] [-v VOLUME...] [-p PORT...] [-e KEY=VAL...] SERVICE [COMMAND] [ARGS...]`